ear/fileio/adm/elements/geom.py

```diff
@@ -27,20 +27,42 @@ def convert_object_position(value):
 
 @attrs(slots=True)
 class ScreenEdgeLock(object):
+    """ADM screenEdgeLock information from position elements
+
+    Attributes:
+        horizontal (Optional[str]): screenEdgeLock from azimuth or X
+            coordinates; must be ``left`` or ``right``.
+        vertical (Optional[str]): screenEdgeLock from elevation or Z
+            coordinates; must be ``top`` or ``bottom``.
+    """
+
     horizontal = attrib(default=None)
     vertical = attrib(default=None)
 
 
 class ObjectPosition(object):
     """Base for classes representing data contained in `audioBlockFormat`
-    `position` elements for Objects."""
+    `position` elements for Objects.
+
+    See Also:
+        :class:`ObjectPolarPosition` and :class:`ObjectCartesianPosition`
+    """
+
     __slots__ = ()
 
 
 @attrs(slots=True)
 class ObjectPolarPosition(ObjectPosition, PolarPositionMixin):
     """Represents data contained in `audioBlockFormat` `position` elements for
-    Objects where polar coordinates are used."""
+    Objects where polar coordinates are used.
+
+    Attributes:
+        azimuth (float)
```

Comment:
In degrees

Reply:
Should be fixed. In general I'd like to rely on correspondence with the ADM though to avoid too much duplication.